### PR TITLE
Increasing weight limit for Lelantus joinsplit tx

### PR DESCRIPTION
**Pull request by levonpetrosyan93**
Depends on #1036

### CODE REVIEW

**Review by a-bezrukov — CHANGES_REQUESTED** [review]

**Review by a-bezrukov — APPROVED** [review]

File: src/consensus/params.h
Comment (levonpetrosyan93) [reply src/consensus/params.h:336]: Yes, correct, we need to merge this PR after merging payload PR.

**Review by psolstice — APPROVED** [review]

**Pull request merged**
